openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -395,13 +396,22 @@ def fake_reverse(*args, **kwargs):
         course_id = 'course-v1:edX+DemoX+Demo_Course'
         return_to = 'info'
 
-        expected_url = (
-            '/enterprise/grant_data_sharing_permissions?course_id=course-v1%3AedX%2BDemoX%2BDemo_'
-            'Course&failure_url=http%3A%2F%2Flocalhost%3A8000%2Fdashboard%3Fconsent_failed%3Dcou'
-            'rse-v1%253AedX%252BDemoX%252BDemo_Course&enterprise_customer_uuid=cf246b88-d5f6-4908'
-            '-a522-fc307e0b0c59&next=http%3A%2F%2Flocalhost%3A8000%2Fcourses%2Fcourse-v1%3AedX%2B'
-            'DemoX%2BDemo_Course%2Finfo'
-        )
+        if six.PY2:
```

Comment:
```
import urllib.parse
expected_parsed = urllib.parse.parse_qs(expected_url.split('/enterprise/grant_data_sharing_permissions?')[1])
actual_parsed = urllib.parse.parse_qs(actual_url.split('/enterprise/grant_data_sharing_permissions?')[1])
        self.assertEqual(expected_parsed, actual_parsed)

Reply:
I like this better, let's parse the URL and compare the components instead of checking the string equality here.

Reply:
got it.